### DISCUSSION TITLE
Experimental Game Settings window, during the same session, the program should remember the position of the list of options rather than always going back to the top of the list

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2690] [Ideas]
In fheroes2, it would be nice that, during the same session, the program remembers the position of the list of options rather than always going back to the top of the list.

So, this:

![Game Setttings list position 02](https://user-images.githubusercontent.com/43583994/106067550-39f1b180-60ff-11eb-8c08-aebbaa85e254.png)

![Game Setttings list position 02](https://user-images.githubusercontent.com/43583994/106067550-39f1b180-60ff-11eb-8c08-aebbaa85e254.png)

Instead of this:

![Game Setttings list position 02](https://user-images.githubusercontent.com/43583994/106067550-39f1b180-60ff-11eb-8c08-aebbaa85e254.png)

![Game Setttings list position 01](https://user-images.githubusercontent.com/43583994/106067561-4413b000-60ff-11eb-83b4-c05f05d52e7e.png)

**Comment by Branikolog** [comment]
It can ease the way people turn on/of options...
If we're going to leave current window design up to release 1.0....

**Comment by LeHerosInconnu** [comment]
This window has been removed from fheroes2.
I close the discussion
